content/en/blog/_posts/2018-06-26-kubernetes-1-11-release-announcement.md

```diff
@@ -24,10 +24,6 @@ In this release, [IPVS-based in-cluster service load balancing](https://github.c
 
 [CoreDNS](https://coredns.io) is now available as a [cluster DNS add-on option](https://github.com/kubernetes/features/issues/427), and is the default when using kubeadm. CoreDNS is a flexible, extensible authoritative DNS server and directly integrates with the Kubernetes API. CoreDNS has fewer moving parts than the previous DNS server, since it’s a single executable and a single process, and supports flexible use cases by creating custom DNS entries. It’s also written in Go making it memory-safe. You can learn more about CoreDNS [here](https://youtu.be/dz9S7R8r5gw).
 
-## Dynamic Kubelet Configuration Moves to Beta
-
-This feature makes it possible for new Kubelet configurations to be rolled out in a live cluster.  Currently, Kubelets are configured via command-line flags, which makes it difficult to update Kubelet configurations in a running cluster. With this beta feature, [users can configure Kubelets in a live cluster](/docs/tasks/administer-cluster/reconfigure-kubelet/) via the API server.
-
```

Comment:
We should leave this _section_ in (it's OK to remove the hyperlink, but leave the text as-is).